draft-ietf-jsonpath-base.md

```diff
@@ -793,7 +793,9 @@ The `index-wild-selector` behaves identically to the `dot-wild-selector`.
 #### Syntax
 {: unnumbered}
 
-The array slice selector has the form `'[<start>:<end>:<step>]'`. It selects elements starting at index `<start>`, ending at &ndash; not including &ndash; `<end>`, while incrementing by `step`.
+The array slice selector has the form `'[<start>:<end>:<step>]'`.
+It selects elements starting at index `<start>`, ending at — but
+not including — `<end>`, while incrementing by `step`.
```

Comment:
This doesn't quite match up with what's actually going on.  Hence [the issue](https://github.com/jsonpath-standard/jsonpath-compliance-test-suite/issues/3) in the compliance test suite I raised.

I was thinking it worked as you've written, but the referenced algorithm doesn't work exactly that way.

Reply:
We didn't discuss this, so I left it for a future PR